### ISSUE TITLE
Low Contrast Ratio for Focus Indicator in 'Latest News' Section

### ISSUE THREAD

**Issue by marvin19** [w3c/w3c-website#600]
**Describe the issue**
When navigating with the keyboard in the "Latest News" section on the front page, the focus indicator has a yellow background. This give me a contrast ratio of 1.4:1 from white text on yellow background.
![Screenshot 2024-01-11 at 08 23 29](https://github.com/w3c/w3c-website/assets/14174575/979a7bc5-0709-4677-8c00-04a3c56c8cca)

**URL**
https://www.w3.org/

**Recommended solution**
Changing the text color to black when focused, and leave it at white when unfocused.
